frontend/packages/console-shared/src/constants/common.ts

```diff
@@ -37,6 +37,7 @@ export const LAST_NAMESPACE_NAME_LOCAL_STORAGE_KEY = `${STORAGE_PREFIX}/last-nam
 export const LAST_PERSPECTIVE_LOCAL_STORAGE_KEY = `${STORAGE_PREFIX}/last-perspective`;
 export const API_DISCOVERY_RESOURCES_LOCAL_STORAGE_KEY = `${STORAGE_PREFIX}/api-discovery-resources`;
 export const COMMUNITY_PROVIDERS_WARNING_LOCAL_STORAGE_KEY = `${STORAGE_PREFIX}/community-providers-warning`;
+export const COMMUNITY_PROVIDERS_WARNING_CONFIGMAP_KEY = `console.communityProvidersWarning`;
```

Comment:
`console` is used across for CM key prefix , will it make sense to keep it in const ?

Reply:
Moved.